### ISSUE TITLE
Utility class's name should end in 'Util'

### ISSUE THREAD

**Issue by Fleshgrinder** [kalessil/phpinspectionsea#126]
```php
final class ValueObject {

    private function __construct() {}
    public static function fromInt() {}

}
```

I highly prefer the Java way of naming methods and we have plenty of those in PHP itself as well (e.g. Intl stuff). However, this inspection is not configurable and only disappears if it encounters a method that starts with `create*`.

Would be great if the current `create*` is extended with `from*`. The `valueOf*` prefix from the JDK does not seem to be widely used in the PHP community and I would ignore it.

😼

**Comment by kalessil** [comment]
This is a minor change, hence dropping into 2.3.2

**Comment by kalessil** [comment]
Added support for valueOf as well, extended test cases to cover mentioned cases.

The strange part is that from* was recognized and the class has been not reported.
@Fleshgrinder : would you please check with a binary from the master and let me know if it's not being fixed?

**Comment by Fleshgrinder** [comment]
Works perfect with the jar from master, thanks a lot. 👍

**Comment by kalessil** [comment]
You are welcome )

**Comment by Fleshgrinder** [comment]
Sorry to post in this issue again, but could we maybe add `new` to the list of exceptions for this one?

Just `new` not `new*`!

**Comment by kalessil** [comment]
@Fleshgrinder : this seems to be project specific, as it none of PHP/Java naming practices.

Before making any decision: is it possible to provide any opensource projects links where this naming is applied to?

**Comment by Fleshgrinder** [comment]
None that I know of because they are all still supporting PHP 5. Would it be possible to expose a RegEx or something?

**Comment by kalessil** [comment]
Not yet, this not supposed to be configurable at all.
If it's just a few classes, I'd suggest suppressing the warning there - just not to expose space for bypassing best practices via any configuration.

**Comment by Fleshgrinder** [comment]
Well, one can always disable an inspection, hence, I do not think that this argument is valid. I for myself would consider any `createFrom*` a violation of best practice because it is _doppelt gemoppelt_ as we say in German (English sounds boring: _done twice_ 😉 ). Hence, I’d definitely want to remove that from my configuration while adding `new` to the list.

**Comment by kalessil** [comment]
Heh, then we need to add configuration here: #154
